lib/node_modules/@stdlib/ndarray/base/to-flippedlr/benchmark/benchmark.js

```diff
@@ -0,0 +1,331 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var isndarrayLike = require( '@stdlib/assert/is-ndarray-like' );
+var baseEmpty = require( '@stdlib/ndarray/base/empty' );
+var empty = require( '@stdlib/ndarray/empty' );
+var pkg = require( './../package.json' ).name;
+var toFlippedlr = require( './../lib' );
+
+
+// MAIN //
+
+bench( pkg+'::1d,base', function benchmark( b ) {
+	var values;
+	var v;
+	var i;
+
+	values = [
+		baseEmpty( 'float64', [ 2 ], 'row-major' ),
+		baseEmpty( 'float32', [ 2 ], 'row-major' ),
+		baseEmpty( 'int32', [ 2 ], 'row-major' ),
+		baseEmpty( 'complex128', [ 2 ], 'row-major' ),
+		baseEmpty( 'generic', [ 2 ], 'row-major' )
+	];
+
+	b.tic();
+	for ( i = 0; i < b.iterations; i++ ) {
+		v = toFlippedlr( values[ i%values.length ] );
+		if ( typeof v !== 'object' ) {
+			b.fail( 'should return an ndarray' );
+		}
+	}
+	b.toc();
+	if ( !isndarrayLike( v ) ) {
+		b.fail( 'should return an ndarray' );
+	}
+	b.pass( 'benchmark finished' );
+	b.end();
+});
+
+bench( pkg+'::1d,non-base', function benchmark( b ) {
+	var values;
+	var v;
+	var i;
+
+	/* eslint-disable object-curly-newline */
+
+	values = [
+		empty( [ 2 ], { 'dtype': 'float64' }),
```

Comment:
```suggestion
		empty( [ 2 ], { 'dtype': 'float64' } ),
```

Applies here and elsewhere in this file. It will require to disable an additional lint rule. This is to ensure consistency with how we sometimes inline objects in `test.ts` files.